Makefile

```diff
@@ -130,17 +132,23 @@ codegen:
 
 stork:
 	@echo "Building the stork binary"
-	@cd cmd/stork && CGO_ENABLED=0 GOOS=linux go build $(BUILD_OPTIONS) -o $(BIN)/stork
+	docker run --rm -v $(shell pwd):/go/src/github.com/libopenstorage/stork  $(DOCK_BUILD_CNT) \
+           /bin/bash -c 'cd /go/src/github.com/libopenstorage/stork/cmd/stork && \
+		   CGO_ENABLED=0 GOOS=linux go build $(BUILD_OPTIONS) -o /go/src/github.com/libopenstorage/stork/bin/stork;'
```

Comment:
Will it not be a problem for "make container" where we do docker build using the dockerfile and there it needs to copy the stork binaries into the container from the host path itself ?